CONTRIBUTING.md

```diff
@@ -252,28 +234,67 @@ examples are run. See the relevant section of the
 
 ### Building the documentation locally
 We recommend that you build and view the documentation website locally, before you push it.
-To do so, first install the requirements for building the documentation:
+To do so, first navigate to `docs/`.
+All subsequent commands should be run from within this directory.
+```sh
+cd docs
+```
+Install the requirements for building the documentation:
 ```sh
-pip install -r docs/requirements.txt
+pip install -r requirements.txt
 ```
 
-Then, from the root of the repository, run:
+Build the documentation:
+
+::::{tab-set}
+:::{tab-item} All platforms
```

Comment:
Not a strong preference, but I wonder whether we should make Unix the first tab across all of these.
That's because the `make` commands are much shorter, so if you are on Linux/Mac, that should be your default. Windows users can still click on the other tab.

But if you disagree, I'm also fine with how it is now.